### ISSUE TITLE
Add tests for portrait and landscape images

### ISSUE THREAD

**Issue by piymis** [rohitk-singh/WhatsApp-Square-DP#11]
Tests for checking correctness of both landscape and portrait modes.
https://github.com/recurser/exif-orientation-examples
http://www.daveperrett.com/articles/2012/07/28/exif-orientation-handling-is-a-ghetto/
We can then trigger the tests using CI.

**Comment by piymis** [comment]
@rohitk-singh by CI, did you mean continuous integration or command interface?

**Comment by rohitk-singh** [comment]
@piymis I meant continuous integration. We can run the test suite after every commit to make sure it doesn't break existing functionality

**Comment by piymis** [comment]
Cool. Do you have something in mind for CI?

**Comment by rohitk-singh** [comment]
Travis CI would do fine. I've enabled CI for this repo on Travis. Let me check if I can get it working

**Comment by rohitk-singh** [comment]
**Update:** We now have TravisCI configured at https://travis-ci.org/rohitk-singh/WhatsApp-Square-DP
All commits and merge requests can now be checked for regression and build tests. Build status would also appear besides commits https://github.com/rohitk-singh/WhatsApp-Square-DP/commits/master

**Comment by piymis** [comment]
Cool!
Mind adding few positive and negative unit tests?

**Comment by rohitk-singh** [comment]
Yeah please do add some. We should probably use pytest/unittest/nose for the tests.

**Comment by piymis** [comment]
Okay let me take a look and come back within a day or two.

**Comment by rohitk-singh** [comment]
Sure! Meanwhile I'll add some if I can figure out how to do that.
